docs/onboarding.rst

```diff
@@ -76,6 +76,12 @@ to access the servers over SSH.
              and only the *Admin Workstation* should have shell access to the
              servers.
 
+.. warning:: The ``app-journalist-aths`` file contains a password for the 
+             authenticated hidden service used by the *Journalist Interface*, 
+             and should not be shared except through the onboarding process. 
+             Make sure to securely delete it from the Transfer Device once 
```

Comment:
Should be specify what securely delete means here? I think this may be necessary for some admins.

Reply:
That's a good point, I don't want to overdo the amount of information in the warning but I'll see if I can adjust the language a bit.